### ISSUE TITLE
bootsy_area is not showing up the editor rather rendering a simple text_area

### ISSUE THREAD

**Issue by hamzaaltaf** [volmer/bootsy#226]
**code in view**
 <% @answer = Answer.new %>
          <%= form_for @answer, :url => { :controller =>"answers", :action =>"create"}  do |f| %>
          <div class="form-group">
            <p><strong>Your answer</strong></p>
            <%= f.bootsy_area(:answer, class: 'form-control').html_safe %><br>

**Answer Modal**
class Answer < ActiveRecord::Base
  include Bootsy::Container
  belongs_to :question
  belongs_to :user
  has_many :replies

  mount_uploader :image, ImageUploader

  # Validations
  validates_associated :user
  validates_associated :question
  validates :answer, presence: true, uniqueness: true, length: { minimum: 5 }
end

browser developer panel show that  "indexOf is not a function" and bootsy area shows up as simple text_area,please let me know how to fix this

**Comment by volmer** [comment]
Can you provide a sample application that reproduces the error?

**Comment by hamzaaltaf** [comment]
can you explain what you mean by this?? you mean the whole rails app?

**Comment by volmer** [comment]
I am asking you to send me a Rails app that reproduces the error, yes. Preferably shared as a GitHub repo. It doesn't have to be the app you are working on, it can be as simple as a barebones app where you see the error happening.

**Comment by hamzaaltaf** [comment]
okay i am going to share the repo with you kindly let me know why it is creating that issue

**Comment by Academy-of-One** [comment]
I am experiencing the exact same problem. Did anything come of this?

**Comment by ManuelFrigerio** [comment]
Having the same problem. I follow the guide in the Readme, restart server, etc.
But it's still showing the old textarea, not Bootsy..

**Comment by volmer** [comment]
@ManuelFrigerio Would you be able to share a sample Rails app in which this problem happens?

**Comment by richardsondx** [comment]
+1 having the same issue
![Booksy issue](https://i.gyazo.com/4c5a6ece4c05b171edfedce44ee37451.gif)

```
Uncaught TypeError: this.$el.modal is not a function
    at new Bootsy.Modal (modal.self-4002063….js?body=1:89)
    at Bootsy.Area.init (area.self-719c9e5….js?body=1:57)
    at HTMLTextAreaElement.<anonymous> (init.self-9c62b2a….js?body=1:22)
    at Function.each (jquery.self-bd7ddd3….js?body=1:371)
    at jQuery.fn.init.each (jquery.self-bd7ddd3….js?body=1:138)
    at Object.Bootsy.init (init.self-9c62b2a….js?body=1:12)
    at init.self-9c62b2a….js?body=1:32
    at dispatch (jquery.self-bd7ddd3….js?body=1:5227)
    at elemData.handle (jquery.self-bd7ddd3….js?body=1:4879)
```

and
![browser error](https://i.gyazo.com/fbc3375b646f4c7a76d2f1126ff190e5.png)

**Comment by volmer** [comment]
I really need a Rails app in order to see what's going on, guys...

**Comment by fippu82** [comment]
@richardsondx @hamzaaltaf @ManuelFrigerio Just a guess: Did you guys check if `/assets/bootsy/modal.js` got loaded correctly? If not, maybe that's a starting point for the analysis.

**Comment by fippu82** [comment]
@hamzaaltaf A little off topic, still, please be careful: The words `modal` and `model` have completely different meaning and have nothing to do with each other! It's dangerous to neglect this. 

Modal is a Bootstrap plugin which displays a dialog box/popup window on top of the current page.

Model is the database layer that manages the application's data, as part of the MVC pattern.

**Comment by fippu82** [comment]
@richardsondx  I see in your animated GIF that there is also a bootstrap load error (bootstrap tooltips). Make sure this clears and see if the problem persists. Bootsy makes use of the Modal plugin of Bootstrap end hence it could be that this fails loading after the first error.

**Comment by adeelejaz** [comment]
I just saw this issue, are you guys using jQuery 3.0? I just created another issue (#258) that sounds very familiar to it.

**Comment by volmer** [comment]
Just merged #259, it could be a potential fix. Feel free to reopen it if the problem persists with the latest master.
